src/ikonScraperInterface.py

```diff
@@ -217,7 +217,7 @@ def checkForOpenings(driver, datesAvailable):
 						# get day of week
 						dayOfWeek = datetime.date(year, month, day).strftime("%A")
 						# send alert
-						emailInterface.sendDateToReserveAlertEmail("jjohnson11096@gmail.com", mountain, monthsToCheck[month], str(day), str(year), dayOfWeek, ikonEmail)
+						emailInterface.sendDateToReserveAlertEmail(sys.argv[1], mountain, monthsToCheck[month], str(day), str(year), dayOfWeek, ikonEmail)
```

Comment:
change sys.argv[1] to "ikonEmail" for readability. This is already a variable at top of file